troposphere/emr.py

```diff
@@ -122,15 +122,76 @@ class EbsConfiguration(AWSProperty):
     }
 
 
+class Constraints(AWSProperty):
+    props = {
+        'MinCapacity': (integer, False),
+        'MaxCapacity': (integer, False)
+    }
+
+
+class CloudWatchAlarmDefinition(AWSProperty):
+    props = {
+        'ComparisonOperator': (basestring, False),
+        'EvaluationPeriods': (positive_integer, False),
+        'MetricName': (basestring, False),
+        'Namespace': (basestring, False),
+        'Period': (positive_integer, False),
+        'Statistic': (basestring, False),
+        'Threshold': (positive_integer, False),
+        'Unit': (basestring, False),
+        'Dimensions': ([KeyValue], False)
```

Comment:
This should be `MetricDimension` though it looks like that's mostly a copy of the KeyValue class: http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-emr-cluster-jobflowinstancesconfig-instancegroupconfig-autoscalingpolicy-constraints-scalingrule-scalingtrigger-cloudwatchalarmdefinition-metricdimension.html